uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/ServerTrustView.qml

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ *  Copyright 2012-2025 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import Esri.ArcGISRuntime.Toolkit.Controller
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+Dialog {
+
+    property ArcGISAuthenticationController controller: null
+
+    title: qsTr("Untrusted Host")
+
+    closePolicy: Popup.NoAutoClose
+
+    header: Label {
+        text: `<h1>${title}</h1>`
```

Comment:
this is neat

Reply:
I take no credit. This is simply ported from the existing SslHandshake view :-)